coops-ui/src/mixins/common-mixin.ts

```diff
@@ -1,18 +1,47 @@
 import { Component, Vue } from 'vue-property-decorator'
 import omit from 'lodash.omit'
+import isEqual from 'lodash.isequal'
 
 /**
  * Mixin that provides some useful common utilities.
  */
 @Component({})
 export default class CommonMixin extends Vue {
   /**
-   * Removes the specified property from an object
+   * Removes the specified property from an object.
    *
-   * @param baseObj The base object
-   * @param prop The property to be removed
+   * @param baseObj The base object.
+   * @param prop The property to be removed.
    */
   omitProp (baseObj: Object, prop: Array<string>): Object {
     return omit(baseObj, prop)
   }
+
+  /**
+   * Removes the specified properties from nested objects.
+   *
+   * @param baseObj The base object.
+   * @param keys The nested object keys which to omit properties from.
+   * @param prop The properties to be removed.
+   */
+  omitProps (baseObj: Object, keys: Array<string>, prop: Array<string>): object {
+    let parsedObj = {}
+    Object.keys(baseObj).forEach(keys => {
```

Comment:
For future, this looks like a textbook example that `map` could solve :)

Reply:
Very cool, i'll look into that, thanks!